prov/efa/test/efa_unit_tests.h

```diff
@@ -5,6 +5,9 @@
 #define EFA_UNIT_TESTS_H
 
 #define _GNU_SOURCE
+
+#define MR_MODE_BITS FI_MR_VIRT_ADDR | FI_MR_ALLOCATED | FI_MR_PROV_KEY | FI_MR_LOCAL
```

Comment:
Let's just make slightly more descriptive, e.g. `MR_BASIC_MODE_BITS`. No strong feelings; `MR_MODE_BITS` is just too vague.